src/Runner.Common/Util/NodeUtil.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.ObjectModel;
+
+namespace GitHub.Runner.Common.Util
+{
+
+    public static class NodeUtil
+    {
+        private const string _defaultNodeVersion = "node16";
+        public static readonly ReadOnlyCollection<string> BuiltInNodeVersions = new(new[] {"node12", "node16"});
+        public static string GetNodeVersion()
+        {
+            var forcedNodeVersion = Environment.GetEnvironmentVariable(Constants.Variables.Agent.ForcedNodeVersion);
```

Comment:
I thought we also are going to control this switch based on the server-side feature flag to some degree.

Reply:
Huge plus 1 to this.  Ultimately, we need progressive rollout.

Reply:
FF passed down in job message to runner ...